tests/unit/lms/models/organization_test.py

```diff
@@ -19,7 +19,7 @@ def test_application_instance_relationship(
     ):
         db_session.flush()
 
-        assert organization.application_instances == application_instances
+        assert set(organization.application_instances) == set(application_instances)
```

Comment:
Sets remove duplicates, so if there's any chance of `organization.application_instances` containing unwanted duplicates the test could pass even though it should fail. `sorted()` is often better for this reason.